scripts/install-build-deps

```diff
@@ -107,6 +107,14 @@ parse_cli_options $@
 (( UID == 0 )) ||
     die 'Error: Please, run this script with "root" privileges'
 
+#
+# Added fix to handle issue EOS-23124
```

Comment:
No need to mention jira for which it is fixed, from commit message we can get such info if needed, only for todo's it can be added.

Reply:
Removed comment.